src/azul/indexer/action_controller.py

```diff
@@ -54,9 +55,15 @@ def _handle_events(self,
                        event: Iterable[SQSRecord],
                        message_handler: Callable[[A, JSON], None]):
         for record in event:
-            message = SQSMessage.from_record(record)
-            log.info('Worker handling message %r, attempt #%i (approx), message ID %s',
-                     message.body, message.attempts, message.id)
+            message: SQSMessage | SQSFifoMessage
+            if self._is_from_fifo_queue(record):
```

Comment:
Can we solve this statically instead? I don't think a concrete subclass is dealing with two types of queues.